OrganisationSample/src/main/java/services/EmployeeExistence.java

```diff
@@ -0,0 +1,12 @@
+package services;
+
+import lombok.Data;
+import models.OrganisationDashBoard;
+
+@Data
+public class EmployeeExistence {
+
+    public boolean isEmployeeExist( String empID) {
```

Comment:
Why an extra class created, just to check existence of employee? It could be in the OrganisationDashboard itself.